src/siren.js

```diff
@@ -57,13 +57,10 @@ var SirenExtension = function(mediaTypes) {
     return mediaTypeSet[type] !==  undefined;
   };
 
-  this.dataParser = function(data, headers) {
-    var ret = _.transform(data.properties, function(res, val, key) {
+  this.dataParser = function (data) {
+    var ret = _.transform(data.properties, function (res, val, key) {
       res.unshift({ name: key, value: val });
     }, []);
-    if (data.title) {
-      ret.unshift({ name: 'title', value: data.title });
-    }
 
     return ret;
```

Comment:
This intermediary variable can probably be removed, since it's a simple return of an immediate value.